packages/synamint-extension-sdk/src/shared/interfaces/actions.ts

```diff
@@ -223,24 +223,8 @@ export class GetInvitationWithDomainParams extends CoreActionParams<JSONString |
     return ECoreActions.GET_COHORT_INVITATION_WITH_DOMAIN;
   }
 }
-
-export class UpdateAgreementPermissionsParams extends CoreActionParams<void> {
```

Comment:
Removing all the commands for updating the permissions I think is a bad thing. You still need to be able to update the permissions for a consent contract